### ISSUE TITLE
content pane shows blank lines as a result of query limit

### ISSUE THREAD

**Issue by rdp** [psequel/psequel#167]
Hello.  If you limit the query results to "25" (the default) and look at the contents of a table, it shows the 25 rows as well as "lots of blank rows" under that.  Which gives a slightly misleading impression that those are "all" the rows, since it's ambiguous with have 25 rows total, in the display.

suggestion/feature request:  remove blank lines beneath the 25, or add an ellipsis below the 25th line somehow or other :)

**Comment by psequel** [comment]
Personally I don't find this is misleading. The footer will say "Row 1 - 25 of [total number of rows]". And the "next page" button is grayed out if there are no more pages.

**Comment by rdp** [comment]
Yeah the "empty grey" rows to me  imply you're at the end of a list
<img width="1197" alt="screen shot 2018-02-13 at 2 02 05 pm" src="https://user-images.githubusercontent.com/12496/36173587-c2a87658-10c6-11e8-94e5-f4e93f24f4c2.png">

it works but barely ambiguous FWIW... (you have to carefully look at a small icon in the bottom right, then it it clear, as it were...)
